core/src/main/scala/org/ensime/indexer/FileWatchers.scala

```diff
@@ -151,28 +151,28 @@ private class ApachePollingFileWatcher(
 
     def fileChanged(event: FileChangeEvent): Unit = {
       if (watched(event)) {
-        if (log.isDebugEnabled())
-          log.debug(s"${event.getFile} was changed")
+        if (log.isTraceEnabled())
+          log.trace(s"${event.getFile} was changed")
         listeners foreach (_.fileChanged(event.getFile))
       }
     }
     def fileCreated(event: FileChangeEvent): Unit =
       if (watched(event)) {
-        if (log.isDebugEnabled())
-          log.debug(s"${event.getFile} was created")
+        if (log.isTraceEnabled())
+          log.trace(s"${event.getFile} was created")
         listeners foreach (_.fileAdded(event.getFile))
       }
     def fileDeleted(event: FileChangeEvent): Unit =
       if (base == event.getFile.getName.getURI) {
-        log.info(s"$base (a watched base) was deleted")
+        log.debug(s"$base (a watched base) was deleted")
```

Comment:
no, quite rare actually. I'm not so fussed about debug guards because it's the default. Trace definitely needs a guard though.